src/database/db_wrapper.go

```diff
@@ -49,6 +49,13 @@ func (r *RowsWrapper) Next() bool {
 	return n
 }
 
+func (r *RowsWrapper) Close() {
+	err := r.Rows.Close()
```

Comment:
🤔 Should't we check if `r.Rows != nil` before trying to close it? (Same in the `RowsxWrapper)

Reply:
I can add this since we had already a bug and since the defer could trick you